Status/Debuffs/Debuff_healing.gd

```diff
@@ -0,0 +1,16 @@
+extends DebuffBase
+class_name Debuff_Healing
```

Comment:
class name is `Debuff_Healing` but filename is `Debuff_**h**ealing` which is not consistent